setup_magento

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
```

Comment:
Why can't these be done from the Dockerfile?

Reply:
Generally this file feels kind of complex. Why do we require these? Can you post a refence link for example?

Reply:
1) These steps are required to setup the magento store, install the Paystack plugin, enable it and compile it.

2) I tried to do these steps in the dockerfile, but they throw errors. I don't have an in-depth understanding of the errors (as i'm no magento expert) but I do believe it's because of dependencies that are not available at build time (I could post a snippet of the stack trace). The workaround I found after doing some research was to run these commands at initialization.